frontend/packages/console-shared/src/hooks/useOperator.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { consoleFetchJSON as coFetchJSON } from '@console/dynamic-plugin-sdk/src/utils/fetch';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+export const useOperator = (
```

Comment:
"operator" may be too vague, we should use more explicit terms as far as kind and plurality of the resource this hook returns.